src/components/Navbar.js

```diff
@@ -1,20 +1,34 @@
-import React from "react";
-import { Link } from "react-router-dom";
+import { Link } from 'react-router-dom';
+import React from 'react';
+// import './styles/Nav.css';
+// import login from '../images/login.png';
```

Comment:
OPTIONAL
_I make it optional but this is a general requirement for every project._
- [ ] To make clean code kindly delete commented code.